### DISCUSSION TITLE
Welcome to vets-design-system-documentation Discussions!

### DISCUSSION THREAD

**Discussion by cvalarida** [department-of-veterans-affairs/vets-design-system-documentation#525] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GnatalieH** [comment]
Hi! I'm Natalie and I'm the new Sr. UX Designer on the VSP Design Systems team.

**Comment by bkjohnson** [comment]
:wave: I'm Brooks, a software engineer on the Design System team.

**Comment by caw310** [comment]
HI. I'm Carol, the product manager on the Design System team.

**Reply by RajuHany420** [reply]
Delete allows device access account device please help me
